meta-resin-common/recipes-support/resin-vars/resin-vars/os-sshkeys.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=OS SSH keys
+Requires=resin-boot.service bind-home-root-.ssh.service
+After=resin-boot.service bind-home-root-.ssh.service
```

Comment:
Shouldn't we add a `Before` statement to ensure this service is started before dropbear?
Something like:
```
Before=dropbear@.service
```

Reply:
@agherzan or @ZubairLK, what is your opinion on my previous comment?

Reply:
Hi @bbinet . Sorry for the late reply. The dropbear will use the generated auth file as soon as it is done. There is no dependency at this point in between these two services but I could see one which makes dropbear depend on this service (similar to the host key service).

Reply:
Thanks @agherzan for your answer.
I'm curious about the other dependency you could see between these 2 services.

Reply:
@agherzan but if the `dropbear_keys_merger` dropbear ExecStartPre script is run before this `os-sshkeys.service`, then the local authorized key available in `config.json` at `.os.sshKeys` path will not be merged into the final `/home/root/.ssh/authorized_keys` file, isn't it?

Reply:
@bbinet The merger runs at each connection and will refresh the available keys as soon as they are available. Either from the config.json translator or from os-config.

Reply:
Ok, thank you! Sorry for my ignorance, I begin with systemd and I didn't know about systemd socket activation which seems to be the reason for the delayed ondemand start of dropbear service, which explains why the merger runs at each connection.

Reply:
Exactly.